src/stirling/source_connectors/tcp_stats/tcp_stats_connector.cc

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "src/stirling/source_connectors/tcp_stats/tcp_stats_connector.h"
+
+#include <arpa/inet.h>
+#include <string>
+#include <utility>
+
+#include "src/common/base/base.h"
+#include "src/common/base/inet_utils.h"
+#include "src/stirling/bpf_tools/macros.h"
+#include "src/stirling/source_connectors/tcp_stats/tcp_stats.h"
+
+OBJ_STRVIEW(tcpstats_bcc_script, tcpstats);
+DEFINE_bool(JsonOutput, true, "Standard output in Json format");
```

Comment:
Just noticed this here - can we remove for now?

Reply:
Deleted